internal/commands/pause_notify.go

```diff
@@ -103,10 +106,9 @@ func PauseNotifyIncidentByDialog(
 
 	pauseNotifyTime.Time = time.Now().AddDate(0, 0, days)
 
-	app.Logger.Info(
+	logWriter.Debug(
```

Comment:
Suggestion: move this to before the days conversion, right after declaring `logWriter`

Reply:
We can't because it depends indirectly on the `days` variable.